modules/shortcodes/slideshare.php

```diff
@@ -91,14 +91,18 @@ function slideshare_shortcode( $atts ) {
 		$player .= " frameborder='" . intval( $attr['fb'] ) . "'";
 	}
 
-	// check the margin width; if not empty, cast as int.
-	if ( ! empty( $attr['mw'] ) || '0' === $attr['mw'] ) {
-		$player .= " marginwidth='" . intval( $attr['mw'] ) . "'";
-	}
+	$is_amp = class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ? true : false;
```

Comment:
I think this could be simplified a bit:

```suggestion
	$is_amp = ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() );
```